public/app/features/search/state/SearchStateManager.ts

```diff
@@ -49,6 +49,7 @@ export class SearchStateManager extends StateManagerBase<SearchState> {
     }
 
     stateManager.setState({
+      query: '',
```

Comment:
I think we need to clear basically all search state. Sorting is still preserved, and that kicks us to search view.

1. Search for folder
2. Set a sort value
3. Click folder
4. Not items in the folder are shown
    - When navigated to folder view, query is cleared but sort is preserved

Reply:
This confuses me a lot. Currently `isSearching` is based on whether filters and queries are available. When should it be active?
- Query or filters available from the URL
- When a filter is changed/updated? This will probably cause weird UI inconsistencies, where `starred` is checked but nothing is listed properly. Should the filters be hidden outside of search view?